controllers/forgotPassword.js

```diff
@@ -0,0 +1,19 @@
+const User = require('../db/models/User')
+const tokenCreator = require('../controllers/token.js')
+const sendEmail = require('../controllers/sendEmail')
+const forgotPasswordEmail = require('../templates/forgotPasswordTemplate')
+
+exports.sendForgotPasswordEmail = (req, res) => {
+  return User.findOne({ email: req.body.email })
```

Comment:
pls move request to query


UserQuery.findUserOne(email) - not needed one?

Reply:
done